doc/GettingStarted.md

```diff
@@ -122,6 +125,64 @@ In addition, Vitess requires the software and libraries listed below.
     ``` sh
     $ sudo apt-get install openjdk-7-jre
     ```
+    
+#### OS X
+
+1.  [Install Homebrew](http://brew.sh/).
+
+2.  On OS X, MySQL 5.6 has to be used, MariaDB doesn't work for some reason yet. It should be installed from Homebrew
+    (install steps are below).
+   
+3.  ZooKeeper is used as lock service.
+
+4.  Run the following commands:
+
+    ``` sh
+    brew install go automake libtool memcached python mercurial git bison curl wget homebrew/versions/mysql56
```

Comment:
I also had to `brew install pkg-config`. Not sure if that's just because my first attempt at running this install line failed partway through due to the permission issues.